### PR TITLE
Update browser support info in intro tutorial

### PR DESCRIPTION
**Pull request by probins**
follow-up to recent change in api docs

### CODE REVIEW

**Comment by ahocevar** [comment]
We don't support IE < 9 at all any more. So the Canvas renderer works in all supported browsers.

**Comment by probins** [comment]
ok, removed that bit.

**Comment by ahocevar** [comment]
Are you sure you pushed the latest change? Still looks wrong to me.

File: doc/tutorials/introduction.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Although Closure library functions are widely used within OL3, none of them are
...
 
 ## Renderers and Browser Support
-The library currently includes three renderers: Canvas, DOM, and WebGL. All three support raster data from tile/image servers, but only the Canvas renderer currently supports vector data. This means that only those browsers that [support Canvas](http://caniuse.com/canvas) can handle vector data. In particular, this excludes Internet Explorer versions before 9, though there is some support for those in the DOM renderer. Clearly, the WebGL renderer can only be used on those devices and browsers supporting WebGL.
+The library currently includes three renderers: Canvas, DOM, and WebGL. All three support both raster data from tile/image servers, and vector data; WebGL however only supports Point vectors. Clearly only those browsers that [support Canvas](http://caniuse.com/canvas) can use the Canvas renderer; this excludes Internet Explorer versions before 9. Equally, the WebGL renderer can only be used on those devices and browsers supporting WebGL.
```
Comment (tsauerwein) [inline doc/tutorials/introduction.md]: Maybe in this context we could also mention that the WebGL renderer does not support labels?

**Comment by probins** [comment]
rephrased to make it clearer, and added @tsauerwein's suggestion

**Pull request merged**